src/connectionManager.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/*
+The connectionManager class is responsible for connecting to Discord webhooks and sending the article to the corresponding webhook.
+*/
+class connectionManager
+{
+    private $hooks = array();
+
+    /* 
+    Adds a webhook to the array of hooks.
+    */
+    public function addHook($hook)
+    {
+        $this->hooks[] = $hook;
+    }
+
+    /*
+    Sends the article to each Discord webhooks.
+    */
+    public function sendArticle($article)
+    {
+        foreach ($this->hooks as $hook) {
+            $msgr = new discordMessenger($hook);
+            $msgr->sendArticle($article);
```

Comment:
Would recommend using full words within variable names, like `$messenger` here. In this case this variable assignment can also be avoided by using the following line since the messager is not reused in any way.
```php
(new discordMessenger($hook)) -> sendArticle($article);
```